lib/Image.php

```diff
@@ -212,7 +223,9 @@ public static function wp_upload_dir() {
 	 */
 	function init( $iid = false ) {
 		if ( !$iid ) { Helper::error_log('Initalized TimberImage without providing first parameter.'); return; }
-		
+		if ( $iid instanceof Image ) {
```

Comment:
You'd think that, right? but when I change it to...

```php
if ( $iid instanceof __CLASS__ )
```

I get: `Parse error: syntax error, unexpected '__CLASS__' (T_CLASS_C) in /srv/www/timber/lib/Image.php on line 226`